pyproject.toml

```diff
@@ -212,6 +212,7 @@ ignore = [
 [tool.pyright]
 # These are tentative
 # include = ["**/*.py"]
-pythonVersion = "3.8" # check no python > 3.8 features are used
-pythonPlatform = "All"
+pythonVersion = "3.10"
```

Comment:
Is this ok, don't we still have to run on 20.04?

Reply:
Maybe I poke that bear tomorrow. Maybe, just maybe I can do better 🤔

Reply:
done, thank you for not letting be lazy